geotrek/tourism/models.py

```diff
@@ -281,6 +281,7 @@ def __unicode__(self):
         return self.name
 
     class Meta:
+        #db_table = 't_b_systeme_reservation'
```

Comment:
Should be moved in its own commit (uncommented, with the corresponding south migration)